internal/transform/operation.go

```diff
@@ -1003,10 +1003,10 @@ func extractOperationDetails(operation xdr.Operation, transaction ingest.LedgerT
 		default:
 			panic(fmt.Errorf("unknown host function type: %s", op.HostFunction.Type))
 		}
-	case xdr.OperationTypeBumpFootprintExpiration:
-		op := operation.Body.MustBumpFootprintExpirationOp()
-		details["type"] = "bump_footprint_expiration"
-		details["ledgers_to_expire"] = op.LedgersToExpire
+	case xdr.OperationTypeExtendFootprintTtl:
+		op := operation.Body.MustExtendFootprintTtlOp()
+		details["type"] = "extend_footprint_ttl"
+		details["extend_to"] = op.ExtendTo
```

Comment:
Since the keys in the object were renamed, how do you think making the updates to the `details` object in BQ is going to go? Curious if we'll be able to drop the old keys, or if we will just have to append the new fields

Reply:
It just appends new fields. Bigquery doesn't seem to support dropping columns within a record. One hacky is to
* Create new record column (details_new)
* Copy old record data into new record column without the expire column (details --> details_new)
* Delete old record column (details)
* Create a record column (empty details)
* Copy data into new record column (details_new --> details)

Reply:
meh, I don't think it's worth the pain to do the drop and recopy (plus that would be pretty expensive) I already don't love the `history_operations` schema, and would prefer if we're making updates to the column to just convert to a `json` blob instead.